MonoGame.Framework/Graphics/GraphicsExtensions.cs

```diff
@@ -364,11 +364,15 @@ public static BlendEquationMode GetBlendEquationMode (this BlendFunction functio
 
 		public static BlendingFactorSrc GetBlendFactorSrc (this Blend blend)
 		{
-			switch (blend) {
+			switch (blend) {
+            case Blend.BlendFactor:
+                return BlendingFactorSrc.ConstantColor;
 			case Blend.DestinationAlpha:
 				return BlendingFactorSrc.DstAlpha;
 			case Blend.DestinationColor:
-				return BlendingFactorSrc.DstColor;
+				return BlendingFactorSrc.DstColor;
+            case Blend.InverseBlendFactor:
+                return BlendingFactorSrc.OneMinusConstantColor;

```

Comment:
Added an exception to the functions I touched, I'll leave better cleanup for later.